vernac/declare.ml

```diff
@@ -2536,12 +2536,18 @@ let solve_obligations ~pm n tac =
   let prg = get_unique_prog ~pm n in
   solve_prg_obligations ~pm prg tac
 
+(** Implements [Solve All Obligations with tac] *)
+
```

Comment:
spacing is confusing, is the comment about the code above or below it?
(not just this comment)

Reply:
I'm not aware of conventions where `.ml` documentation is after a function, but that's ok also to not skip a line.

I kept one though for `admit_obligations`: the comment covers both `admit_prog` and `admit_obligations` but not keeping the line between the comment and `admit_prog` would let think it applies only to `admit_prog` (I would say).